### PR TITLE
remove moment-timezone from website bundle 

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
- moment-timezone added to handle global downtime notification in january
- module size is too big to keep in bundle 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -265,6 +264,7 @@
     "metalsmith-permalinks": "^0.5.0",
     "metalsmith-sitemap": "^1.0.0",
     "moment": "^2.15.1",
+    "moment-timezone": "^0.5.27",
```
Comment (rianfowler) [inline package.json]: we still use moment-timezone in our build process

File: src/platform/monitoring/DowntimeNotification/actions/index.js [hunk 2]
```diff
@@ -54,25 +50,10 @@ export function dismissDowntimeWarning(appTitle) {
...
 
-export function getScheduledDowntime(downtimeWindow = scheduledDowntimeWindow) {
+export function getScheduledDowntime() {
   return async dispatch => {
     dispatch({ type: RETRIEVE_SCHEDULED_DOWNTIME });
 
-    // create global downtime data if feature toggle is enabled and if
-    // current time is in downtime window
-    // default to empty array
-    const { downtimeStart, downtimeEnd } = downtimeWindow;
```
Comment (rianfowler) [inline src/platform/monitoring/DowntimeNotification/actions/index.js]: oddly removing all of this code did not break any of our unit tests.

**Review by billfienberg — APPROVED** [review]
I'm surprised that deleting this code didn't break any tests, but the change looks safe to me.

File: src/platform/monitoring/DowntimeNotification/actions/index.js (hunk 2 above)
Comment (bkjohnson) [reply src/platform/monitoring/DowntimeNotification/actions/index.js]: If I remember correctly, I think it is because you skipped some of the added tests.

**Pull request merged**